### ISSUE TITLE
Incorrect arity for plumbing threading macros fn-> and fn->>

### ISSUE THREAD

**Issue by ikitommi** [cursiveclojure/cursive#771]
```clojure
((fn-> (assoc :a 1)) {})
```

complains about Incorrect Arity 2. Same for `fn->>`

**Comment by cursiveclojure** [comment]
Yeah, I don't have a good solution for this yet, because the implicit argument doesn't appear anywhere in the source code at all. I'm still not sure how to handle this, but I'll fix it when I come up with something.

**Comment by alexanderkiel** [comment]
I have the same problem. `fn->` is just different from `->` as it doesn't take a "normal" first argument.

**Comment by wnr** [comment]
I too have issues with this. Is there a workaround?
